docs/selectors.md

```diff
@@ -0,0 +1,14 @@
+# Selectors
```

Comment:
Nice addition.  Looking at this alongside other docs it's becoming clear the file organization I began here is not ideal.  Right now its unclear what files are documenting what.  This documentation has a different context from that documenting DSL features and our docs should convey that.  I'm thinking either one large file with broader subsections (kubernetes-ipfs commands, The DSL, Kubernetes Config, etc) or perhaps separate files for each broader subsection.

Reply:
On that note it would be good to have more background on the kubernetes config so that this section will make sense to people without a kubernetes background.

Reply:
@FrankPetrilli both of these comments should be addressed before merging: 
1. Consolidate docs into one file or a few logically laid out files to make documentation more approachable
2. Preface this specific section with background on the kubernetes config file that the selector key value pair goes into.  Could be as simple as a link to existing kubernetes docs or a few concise sentences following a header.

Reply:
That sounds along the right track.  I think the best answer is that I don't have the kubernetes context that you do to get much more specific.  The labels exist in a file that I don't know much about.  It would be great if the docs introduced me to this section without assuming I knew what that file is called, what it is used for and what its format is.  imo the best way to do this is with the structure suggested in (1) including this section as a subsection of "the kubernets yaml definition"

Reply:
or a name you think fits better :)